src/app/themes.tsx

```diff
@@ -10,53 +10,121 @@ import {
+    light: true,
+    main: {
+        background: ["#fdfdfd", "#eeeeee", "#eeeeee", "#eeeeee"],
+        foreground: ["#000000", "#2d2d2d", "#383838", "#4d4d4d"],
+        largeForeground: ["#000000", "#2d2d2d", "#383838", "#4d4d4d"],
+        border: "#d5d5d5",
+    },
+    primary: {
+        background: ["#a3e0b0", "#7fd495", "#02985a", "#05663f", "#154d30"],
+        foreground: ["#000000", "#000000", "#000000", "#f2f2f2", "#f2f2f2"],
+        largeForeground: ["#000000", "#000000", "#f2f2f2", "#f2f2f2", "#f2f2f2"],
+    },
+    error: "#ff624e",
+    shadow: "#e2e2e2",
+    rainbow: {
+        red: {
+            background: ["#fa9189", "#fb685a", "#fc2a2a"],
+            foreground: ["#000000", "#000000", "#000000"],
+            largeForeground: ["#000000", "#000000", "#f2f2f2"],
+        },
+        orange: {
+            background: ["#fcae7c", "#ff9753", "#fa6400"],
+            foreground: ["#000000", "#000000", "#000000"],
+            largeForeground: ["#000000", "#000000", "#000000"],
+        },
+        yellow: {
+            background: ["#ffd868", "#ffcb1e", "#ffbb00"],
+            foreground: ["#000000", "#000000", "#000000"],
+            largeForeground: ["#000000", "#000000", "#000000"],
+        },
+        lightGreen: {
+            background: ["#b4f5b3", "#7ceb98", "#1ae669"],
+            foreground: ["#000000", "#000000", "#000000"],
+            largeForeground: ["#000000", "#000000", "#000000"],
+        },
+        darkGreen: {
+            background: ["#8ecbb4", "#66b79b", "#36d5a7"],
+            foreground: ["#000000", "#000000", "#000000"],
+            largeForeground: ["#000000", "#000000", "#000000"],
+        },
+        blue: {
+            background: ["#acd5df", "#7bbafc", "#5b74f1"],
+            foreground: ["#000000", "#000000", "#000000"],
+            largeForeground: ["#000000", "#000000", "#f2f2f2"],
+        },
+        purple: {
+            background: ["#ceaae9", "#b18ff9", "#af25ff"],
+            foreground: ["#000000", "#000000", "#000000"],
+            largeForeground: ["#000000", "#000000", "#f2f2f2"],
+        },
+        grey: {
+            background: ["#d7d7d7", "#959595", "#808080"],
+            foreground: ["#000000", "#000000", "#000000"],
+            largeForeground: ["#000000", "#000000", "#f2f2f2"],
+        },
+        brown: {
+            background: ["#bfaba2", "#a49589", "#87735b"],
+            foreground: ["#000000", "#000000", "#000000"],
+            largeForeground: ["#000000", "#000000", "#f2f2f2"],
+        },
+    },
+};
+
+export const darkTheme: customTheme = {
+    ...lightTheme,
+    light: false,
+    main: {
+        background: ["#232323", "#2d2d2d", "#2d2d2d", "#2d2d2d"],
+        foreground: ["#d9d9d9", "#d9d9d9", "#B5B5B5", "#969696"],
+        largeForeground: ["#f2f2f2", "#ededed", "#B5B5B5", "#969696"],
+        border: "#5c5c5c",
+    },
+    primary: {
+        background: lightTheme.primary.background.slice().reverse(),
```

Comment:
This seems unnecessary, imo it would be easier to use should the colours be visible, instead of having to find `lightTheme.primary.background`

Reply:
I agree maybe it is nice to keep light and dark mode a bit more independent, rather than just purely the reversed. But currently it works though so maybe it is possible to just keep it like this.

In terms of readability, it doesn't seem too big an issue as it simply means the color palette is reversed between light and dark